### PR TITLE
T18984

### PR DESCRIPTION
**Pull request by cosimoc**
https://phabricator.endlessm.com/T18984

### CODE REVIEW

**Comment by smspillaz** [comment]
This looks good to me - tested the code and it works fine. Thanks.

File: gnome-initial-setup/pages/language/gis-language-page.ui [hunk 1]
```diff
@@ -6,33 +6,54 @@
...
-            <property name="margin_bottom">18</property>
-            <property name="width_request">400</property>
+            <property name="valign">fill</property>
+            <child>
+              <object class="GtkImage" id="logo">
+		<property name="visible" bind-source="GisLanguagePage" bind-property="small-screen" bind-flags="invert-boolean|sync-create"/>
+		<property name="can_focus">False</property>
+		<property name="margin_top">24</property>
+		<property name="pixel_size">96</property>
+		<property name="icon_name">start-here-symbolic</property>
```
Comment (smspillaz) [inline gnome-initial-setup/pages/language/gis-language-page.ui]: Hmm, actually, I didn't see this icon appear in the right place when I ran g-i-is. Is something missing on my system?

Comment (smspillaz) [reply gnome-initial-setup/pages/language/gis-language-page.ui]: Here is what I see for what its worth:

screenshot from 2017-09-02 13-52-56

Comment (cosimoc) [reply gnome-initial-setup/pages/language/gis-language-page.ui]: Yeah, we override that icon on the `GtkImage` here: https://github.com/endlessm/gnome-initial-setup/commit/92dd4344c60f7fe7822efbcbc512aa6569cde37f

**Pull request merged**